regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -39,8 +39,7 @@ This contains all of the styles specific to the TOC
     a:link,
     a:visited {
         color: @black;
-        .border-bottom-light (@width: 1px);
-        text-decoration: none;
+        .border-bottom-light (@width: 0px);
```

Comment:
ah good call, actually this removes the bottom border for something else (the items in the table of contents) and I didn't mean to make that change in this PR